### ISSUE TITLE
Importing setPublicPath

### ISSUE THREAD

**Issue by pedro-otero** [joeldenning/systemjs-webpack-interop#5]
This line

```javascript
import { setPublicPath } from "systemjs-webpack-interop";
```

Does not work for me. I get a `TypeError: Object(...) is not a function` at the exact line where I use the function.

The only way I can make that function work is by doing it like this:

```javascript
import { setPublicPath } from 'systemjs-webpack-interop/src/public-path-system-resolve';
```

**Comment by joeldenning** [comment]
What bundler are you using? systemjs-webpack-interop has two libraries in it -- a build time node library for webpack and a runtime browser library for systemjs. It relies on the `module` field in the package.json for the browser to work, and the `main` field in the package.json for node to work.

Webpack and rollup both support using the `module` property for browser stuff, but not all tools do. Notably, jest does not, but also there are likely other tools that do not.

What tool are you using? My guess is that it likely doesn't support the `module` property.

**Comment by pedro-otero** [comment]
@joeldenning webpack

I don't remember changing anything in my package.json. Should it be obvious or be included in the docs?

**Comment by joeldenning** [comment]
Webpack should respect the `module` field in the package.json:

https://github.com/joeldenning/systemjs-webpack-interop/blob/594af750158f3f67ea7a8b89762b8d2779910dca/package.json#L6

[Published package.json](https://unpkg.com/browse/systemjs-webpack-interop@1.1.2/package.json)

Webpack uses [enhanced-resolve](https://github.com/webpack/enhanced-resolve) to do so. I'm not sure why it wouldn't be working for you - this project works well for others.

What version of webpack are you using?

**Comment by joeldenning** [comment]
I published changes in 2.0.0 to simplify this and make it work in all bundlers and all situations. I am closing this issue as a result. Let me know if you still run into issues.

Details:
https://github.com/joeldenning/systemjs-webpack-interop/releases/tag/v2.0.0
